code/modules/reagents/chemistry/reagents.dm

```diff
@@ -109,16 +133,32 @@ GLOBAL_LIST_INIT(name2reagent, build_name2reagent())
 		if(amount >= 0.5)
 			exposed_mob.reagents.add_reagent(type, amount)
 
+	if(iscarbon(exposed_mob) && acidpwr > 0)
+		var/mob/living/carbon/exposed_carbon = exposed_mob
+		if(methods & INGEST)
+			exposed_carbon.adjustBruteLoss(min(6 * acidpwr, round(reac_volume,0.1) * acidpwr))
+			return
+		if(methods & INJECT)
+			exposed_carbon.adjustBruteLoss(1.5 * min(6 * acidpwr, round(reac_volume,0.1) * acidpwr))
```

Comment:
I am pretty sure this is a mistake, but this was based off of `toxpwr` before. The highest acid's `toxpwr` was 3. The highest `acidpwr` is... 42 as far as I can tell. 378 `BRUTE` damage half-empty syringe go brrrrr.